images/libvirtd/Dockerfile

```diff
@@ -13,6 +13,15 @@ RUN yum install -y \
 COPY augconf /augconf
 RUN augtool -f /augconf
 
+# Faking qemu-kube
+# qemu-kube originally requires docker and sudo - also
+# access to docker service (which is running on the host)
+# modifying this image would be possible, but it's not
+# essential for the demo.
+# It should be obsoleted anyway once the virt-handler
+# populates the cgroup.
```

Comment:
That's another step.

I just mentioned cgroups in the comment, because that is what libvirtd can do today, we just don't do it in the handler righ tnow.

namespaces are not support - and probably will not be - because it has security implications (need to follow up on this one).